cli/command.go

```diff
@@ -27,8 +27,16 @@ import (
 	"strings"
 
 	"github.com/mattn/go-isatty"
+	"github.com/posener/complete/v2"
+	"github.com/posener/complete/v2/predict"
 )
 
+// isCompletionRequest returns true if the invocation is a completion request,
+// or false otherwise.
```

Comment:
nit: could you expand the comment the say these are env vars recognized by [github.com/posener](https://github.com/posener/complete)? It was not obvious to me initially.

Reply:
Done.